tests/jobserver/views/test_job_requests.py

```diff
@@ -143,7 +142,6 @@ def test_jobrequestdetail_with_superuser(rf, superuser):
     response = JobRequestDetail.as_view()(request, pk=job_request.pk)
 
     assert response.status_code == 200
```

Comment:
Do we need some other "superusery" test here to replace the one we're losing?

Reply:
No, even better, we can remove more code 🎉 (rebased into original commit)